csi/moac/default.nix

```diff
@@ -43,7 +43,7 @@ result // rec {
   #    of layering.
   buildImage = pkgs.dockerTools.buildLayeredImage {
     name = "mayadata/moac";
-    tag = "latest";
+    tag = "develop";
```

Comment:
I'm not sure it is necessary to make such change in the repo. You can set it to whatever you want when building your private image, but there is no reason to change it for everyone.

Reply:
I see what you mean now. Looks good 👍